app/posts/personalised-prevention-platform/2025/04/2025-04-21-alpha.md

```diff
@@ -0,0 +1,253 @@
+---
+title: "Onboarding users"
+description: "Exploring how people might be introduced to the service using prototypes"
+date: 2025-04-21
+tags:
+- alpha
+- prototyping
+---
+
+The personalised prevention platform (PPP) team has moved into an alpha phase.
+
+<figure>
+    <blockquote class="govuk-inset-text govuk-!-margin-left-0 govuk-!-padding-left-6" cite="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">
+      <p>Alpha is where you try out different solutions to the problems you learnt about during discovery.</p>
+      <p>Spend alpha building prototypes and testing different ideas. And do not be afraid to challenge the way things are done at the moment: alpha is a chance to explore new approaches.</p>
+      <p>You do not have to prototype the user’s entire wider journey.</p>
+      <p>You might not even want to prototype all of the transaction or element you’re working on: often it makes sense just to focus on the areas you think will be most challenging. This lets you do the minimum you need to test your riskiest assumptions.</p>
+    </blockquote>
+    <figcaption>From the <a href="https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works">GOV.UK Service Manual</a></figcaption>
+</figure>
```

Comment:
I think you could [probably use markdown here](https://x-govuk.github.io/govuk-eleventy-plugin/markdown/#blockquotes)?

Reply:
Question on this, does it matter? Not being shirty, more that I want the "MDN pattern" of the figcaption and using MD won't get me that (is that right?)